.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
```

```suggestion
    branches: [ "master" ]
```
Is there a reason to change this from the default Scala configuration (which is just `branches: ["master"]`?